codespell_lib/data/dictionary.txt

```diff
@@ -30598,6 +30605,7 @@ wnat->want, what,
 wnated->wanted
 wnating->wanting
 wnats->wants
+wnen->when
```

Comment:
Suggest Wen name too.

Reply:
conflicts with https://github.com/luzpaz/codespell/blob/d8c56c34fa8641b75dbcd6341778af626059cc2c/codespell_lib/data/dictionary_names.txt#L11

Reply:
Added anyway in 8088c43

Reply:
Fixed in 036cf1db2f2cc62e4c440527807d91a5382f07b8